code/__DEFINES/achievements.dm

```diff
@@ -10,6 +10,12 @@
 #define MEDAL_THANKSALOT 		"The Best Driver"
 #define MEDAL_HELBITALJANKEN	"Hel-bent on Winning"
 #define MEDAL_MATERIALCRAFT 	"Getting an Upgrade"
+#define MEDAL_DISKPLEASE	 	"Disk, Please!"
+#define MEDAL_GAMER			 	"My Watchlist Status is Not Important"
```

Comment:
This is above 32 characters isn't it. 
I probably should have made that column bigger but anyway.
I'd generally recommend to use simple tags here and full name in the datum.

Reply:
Oh, wasn't aware of that, alright

Reply:
While you're at it could you put a comment above this section about this.